README.md

```diff
@@ -72,7 +72,7 @@ The **SpendSmart** keeps track of all your expences associated with catogories.
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- The Live Demo is available [here](https://budget-app-riuo.onrender.com/)
+- The Live Demo is available [here](https://budget-app-pkds.onrender.com/)
```

Comment:
- Hey, Your [deployment link](https://budget-app-pkds.onrender.com/)
 is currently broken and not working. Below is the image. Please, could you work on your deployment?

*deployment not working*
![image](https://github.com/yusufyusie/budget-app/assets/59397159/4c35f46b-9a12-4b6a-8060-9559d3aaec73)

Reply:
- Hey, as a requirement for this project, you should deploy your app, this is a key requirement you must implement to have your project approved. But, you're insisting on that your project be reviewed locally. 

*deployment requirement*
![image](https://github.com/yusufyusie/budget-app/assets/59397159/78eea7a2-0b37-4a92-83fa-400892161691)

- I'm sorry, I'd still have to request changes. I sent you a message on slack, but you didn't respond. I hope you find a way to fix it.